### ISSUE TITLE
When I set up my resume_ckpt and resume_D_ckpt and test_ckpt paths according to your config file, and start my second training, the system reports  errors

### ISSUE THREAD

**Issue by RecruitXin** [bobo0303/LIGHTWEIGHT-IMAGE-INPAINTING-BY-STRIPE-WINDOW-TRANSFORMER-WITH-JOINT-ATTENTION-TO-CNN#2]
When I set up my resume_ckpt and resume_D_ckpt and test_ckpt paths according to your config file, and start my second training, the system reports  errors：

Try load VGG mdoel for Perceptual_loss
D:\Anaconda3-install\install\envs\cnn\lib\site-packages\torch\optim\lr_scheduler.py:129: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate
  warnings.warn("Detected call of `lr_scheduler.step()` before `optimizer.step()`. "
D:\Anaconda3-install\install\envs\cnn\lib\site-packages\torch\optim\lr_scheduler.py:508: UserWarning: To get the last learning rate computed by the scheduler, please use `get_last_lr()`.
  warnings.warn("To get the last learning rate computed by the scheduler, "
Please tell me how to solve the above problem, thank you

**Comment by bobo0303** [comment]
I discovered this problem during my research, but it seemed to have no impact on training and results, so I did not optimize this warning.
